views/projects.tpl

```diff
@@ -35,7 +35,7 @@
                         action: function(e, dt, node, config) { set_search(""); }
                     },
                 ],
-                "order": [[5, "desc"], [19, "desc"], [0, "asc"], [1, "asc"]],
+                "order": [[0, "asc"], [5, "desc"], [19, "desc"], [1, "asc"], [2, "asc"]],
```

Comment:
As you're adding a column at the beginning, all other indices should be incremented if we want to keep the rest of the sorting logic:

```suggestion
                "order": [[0, "asc"], [6, "desc"], [20, "desc"], [1, "asc"], [2, "asc"]],
```